GirafAPI/Mapping/UserMapping.cs

```diff
@@ -35,7 +35,7 @@ public static UserDTO ToDTO(this GirafUser user)
     {
         return new UserDTO(
             user.Id,
-            user.Email,
+            user.Email!,
```

Comment:
Same thing

Reply:
The front-end team should at this point, made sure that a bad request hasn't been sent(e.g missing information like email)

Reply:
Validation shouldn't happen at just the frontend. It's the job of both the frontend, backend and for that sake the database schema to ensure that data follows the business logic of the application. I am on the frontend team and our hope is that we follow your rules and try to make a nice UX where the user doesn't attempt to submit invalid data. However there is no guarantee on your part that this is the case. Moreover, the underlying model might change in the future, which might make the frontend validation incorrect.